Update announcements-10-16-0.md (#152)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/announcements-10-16-0.md

As a recommended mechanism to integrate Linux-based devices into Cumulocity IoT we advise you to use [thin-edge.io](https://thin-edge.io/), which is actively developed by Software AG and open-source community partners. For customers with enterprise-grade support & maintenance needs, a commercial variant (Cumulocity IoT Thin Edge) can be purchased.

### Documentation

#### Planned

> The German version of the *User guide* (*Benutzerhandbuch*) is deprecated. In a future release we plan to remove the German *User guide* from the documentation website and focus on providing a high-quality, up-to-date English user documentation. 

Reviewer: maybe "plan to" -> "are going to" (a little stronger, it'll definitely happen)
"focus on providing" sounds a bit like we don't have a high-quality, up-to-date English user documentation yet :)
maybe "focus on providing ..." -> "focus on our English user documentation"

> The German version of the *User guide* (*Benutzerhandbuch*) is deprecated. In a future release we plan to remove the German *User guide* from the documentation website and focus on providing a high-quality, up-to-date English user documentation. 

Author: ```suggestion
The German version of the *User guide* (*Benutzerhandbuch*) is deprecated. In a future release we are going to remove the German *User guide* from the documentation website and focus on our high-quality, up-to-date English user documentation. 
```
